content/en/docs/Writing policies/exceptions.md

```diff
@@ -166,3 +167,187 @@ spec:
             - resources:
                 names: "?*"
 ```
+
+## Pod Security Exemptions
+
+Kyverno policies can be used to apply Pod Security Standards profiles and controls via the [validate.podSecurity](/docs/writing-policies/validate/#pod-security) subrule. However, there are cases where certain Pods need to be exempted from these policies. For example, a Pod may need to run as `root` or require privileged access. In such cases, a PolicyException can be used to define an exemption for the Pod through the `podSecurity{}` block. It can be used to define controls that are exempted from the policy.
+
+Given the following  policy that enforces the latest version of the Pod Security Standards restricted profile in a single rule across the entire cluster.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: psa
+spec:
+  background: true
+  validationFailureAction: Enforce
+  rules:
+  - name: restricted
+    match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+    validate:
+      podSecurity:
+        level: restricted
+        version: latest
+```
+
+There is a case where all Pods exist in the `delta` Namespace needs to run as a root. A PolicyException can be used to exempt all Pods whose namespace is `delta` from the policy by excluding the `runAsNonRoot` control.
+
+```yaml
+apiVersion: kyverno.io/v2beta1
+kind: PolicyException
+metadata:
+  name: pod-security-exception
+  namespace: policy-exception-ns
+spec:
+  exceptions:
+  - policyName: psa
+    ruleNames:
+    - restricted
+  match:
+    any:
+    - resources:
+        namespaces:
+        - delta
+  podSecurity:
+    - controlName: "Running as Non-root"
+```
+
+The following Pod satisfies all controls in the restricted profile except the `Running as Non-root` control but it matches the exception. Hence, it will be successfully created.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-pod
+  namespace: delta
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    args:
+    - sleep
+    - 1d
+    securityContext:
+      seccompProfile:
+        type: RuntimeDefault
+      runAsNonRoot: false
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+```
+
+PolicyExceptions `podSecurity{}` block has the same functionality as the [validate.podSecurity.exclude](/docs/writing-policies/validate/#exemptions) block in the policy itself. They can be used to exempt controls that can only be defined in the container level fields.
+
+For example, the following PolicyException exempts the containers running either the `nginx` or `redis` image from following the Capabilities control.
+
```

Comment:
(Here and below) I know you wrote this before the blog, but I would recommend using the case I wrote about [here](https://kyverno.io/blog/2024/02/04/securing-services-meshes-easier-with-kyverno/#enhancements-in-112) to illustrate this functionality as it is so common.

Reply:
Done.